build/release/release-procedure.adoc

```diff
@@ -0,0 +1,96 @@
+= Release Procedure
+
+== Projects and Artifacts
+Following sections describe what projects are part of the OptaPlanner release procedure and their resulting
+release artefacts.
+
+=== OptaPlanner
+
+* JAR files hosted on Maven Central
+* https://download.jboss.org/optaplanner/release/7.44.0.Final/optaplanner-distribution-7.44.0.Final.zip[Distribution.zip]
+available via a link on the optaplanner-website
+* https://docs.optaplanner.org/7.44.0.Final/optaplanner-docs/html_single/index.html[Documentation]
+available via a link on the optaplanner-website
+* XML Schema Definition files for both the https://www.optaplanner.org/xsd/solver/[Solver]
+and the https://www.optaplanner.org/xsd/benchmark/[Benchmark] available on the optaplanner-website
+
+=== OptaPlanner Quickstarts
+
+No build artifacts are required; as a result of the release, the development branch must be rebased onto
+the stable branch and must point to the released final version of OptaPlanner.
+
+=== OptaWeb Employee Rostering and Vehicle Routing
+
+* https://www.optaplanner.org/download/download.html[Distribution.zip] available via a link on the optaplanner-website
+* https://www.optaplanner.org/learn/documentation.html[Documentation] available via a link on the optaplanner-website
+
+=== OptaPlanner Website
+The website contains https://github.com/kiegroup/optaplanner-website/blob/master/_config/pom.yml[links]
+to the distribution.zip files and documentation (see above). To update the documentation, and the distribution.zip,
+upload them to the JBoss file mgmt and then, open a PR changing the links.
+
+For the OptaPlanner project, there is a https://github.com/kiegroup/optaplanner/blob/master/build/release/upload_distribution.sh[script]
+uploading the documentation and distribution.zip. The release pipeline calls the script and updates also
+the links on the website. For the Optawebs, the upload has to be done manually at this moment.
+
+== Versioning
+
+=== Project Version
+
+The first version released by this procedure is going to be 8.0.0.Final. After the release, the master branch
+upgrades to the version 8.1.0-SNAPSHOT. The next release is 8.1.0.Final
```

Comment:
I think we can make this generic, as in X, X+1 etc.
The sentence you use here can serve as an example.